File: help/eng_gb/exercicios_da_ficha_3/Ficha03_012/Ficha03_012.php
<?php
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Nome : " ;
    $nome1 = isset($_POST["$nome1"])? $_POST["$nome1"] : isset($_GET["$nome1"]) ? $_GET["$nome1"] : 0;
    echo $nome1 . "<br/>";// print input 
    echo  "Nome : " ;
    $nome2 = isset($_POST["$nome2"])? $_POST["$nome2"] : isset($_GET["$nome2"]) ? $_GET["$nome2"] : 0;
    echo $nome2 . "<br/>";// print input 
    echo  "Nomes Ordenados" ;
    echo  "\n" . ( min( $nome1 , $nome2 )  ) ;
    echo  "\n" . ( max( $nome1 , $nome2 )  ) ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	Tue, 2022 Sep 13 21:57

?>

File: help/eng_gb/exercicios_da_ficha_3/Ficha03_012/Ficha03_012.c
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
#define min(x, y) (((x) < (y)) ? (x) : (y))
#define max(x, y) (((x) > (y)) ? (x) : (y))
//Main Function
int main() {
    printf("Nome : ");
    char nome1[256];
    fgets(nome1,256, stdin);
    printf("Nome : ");
    char nome2[256];
    fgets(nome2,256, stdin);
    printf("Nomes Ordenados");
    printf("\n%s ",min( nome1 , nome2 ) );
    printf("\n%s ",max( nome1 , nome2 ) );
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Tue, 2022 Sep 13 21:57
